spiceaidocs/docs/components/data-connectors/postgres/index.md

```diff
@@ -16,31 +16,100 @@ datasets:
 
 :::warning[Limitations]
 
-- The Postgres federated queries may result in unexpected result types due to the difference in DataFusion and Postgres size increase rules. Please explicitly specify the expected output type of aggregation functions when writing query involving Postgres table in Spice. For example, rewrite `SUM(int_col)` into `CAST (SUM(int_col) as BIGINT`.
+The Postgres federated queries may result in unexpected result types due to the difference in DataFusion and Postgres size increase rules. Please explicitly specify the expected output type of aggregation functions when writing query involving Postgres table in Spice. For example, rewrite `SUM(int_col)` into `CAST (SUM(int_col) as BIGINT`.
 
 :::
 
 ## Configuration
 
+### `from`
+
+The `from` field takes the form `postgres:my_table` where `my_table` is the table name in the SQL server to read from.
```

Comment:
```suggestion
The `from` field takes the form `postgres:my_table` where `my_table` is the table identifer in the PostgreSQL server to read from.
```

We should also document that we can select from `database.schema.table`

Reply:
Fixed!